lib/Dialect/MSFT/MSFTPasses.cpp

```diff
@@ -184,6 +185,26 @@ struct PhysicalRegionOpLowering : public OpConversionPattern<PhysicalRegionOp> {
 };
 } // anonymous namespace
 
+namespace {
+/// Simply remove hw::GlobalRefOps for placement when done.
+struct GlobalRefOpLowering : public OpConversionPattern<hw::GlobalRefOp> {
```

Comment:
You might check if there's an upstream pass which just removes ops. Alternatively, I think it'd be reasonable for ExportVerilog to just ignore these. This must be a common thing to do, so you might get with the SiFive guys to see what they plan on doing.

Reply:
I don't know if there is a generic pattern or pass to remove these. I believe the plan (from the original PR) is to ultimately support these in ExportVerilog as something you can refer to in a verbatim node. But that isn't really relevant to us here: after we do ExportQuartusTcl, we have no need for these anchors, but we can't leave them in the IR, because the global ref verifier is going to verify that every op down the path has the correct attribute.